crates/keyvalue/src/implementors/awsdynamodb.rs

```diff
@@ -44,8 +45,9 @@ impl AwsDynamoDbImplementor {
     /// }
     /// ```
     pub async fn new(_slight_state: &BasicState, name: &str) -> Self {
-        let shared_config = aws_config::load_from_env().await;
-        let client = Client::new(&shared_config);
+        let region = RegionProviderChain::default_provider().or_else("us-west-2");
+        let config = from_env().region(region).load().await;
```

Comment:
I wish we could convert this to use our runtime configs like every other crate...

Reply:
this is a great point. We may have to set the environment variables for the running process from the slightfile and use `from_env()`

Reply:
Fixed. Could you PTAL?